### DISCUSSION TITLE
Issues-from-self count is incorrect

### DISCUSSION THREAD

**Discussion by Nixinova** [lowlighter/metrics#855] [Q&A]
Metrics says my issues-from-self count is 0:

<table>
<td><br><br><br><br>

**&rarr;**

</td>
<td>

![image](https://user-images.githubusercontent.com/42429413/152915699-a0fc1201-64d1-43a9-bc84-7f0feffb29f1.png)

</td>
</table>

However I do have an open issue on my repo created by myself here:

github.com\Nixinova\Wikity\issues\1

**Comment by lowlighter** [comment]
Will dig later on it

But one guess would be that the action will try to list the users with push access to know where to put the issue, but this api call actually requires additional scopes (probably `repo` scope).

If that's the issue unfortunately it won't be possible to do much better 😕 
However, in this specific case, it's kind of implied that owner always has push access, so we could make something with it and mitigate a bit the issue

**Comment by Nixinova** [comment]
My pull requests count are also off by 1:
<img width="197" alt="image" src="https://user-images.githubusercontent.com/42429413/155411802-172ae7e4-3f6b-48a9-9b9d-e2c7fbc7a9e4.png">
I have 1 draft & 0 open but this says I have 2 drafts & 1 open

**Reply by lowlighter** [reply]
I haven't looked yet into #859, but from what I remember there were stuff counted twice because the logic was inverted.

It may be related to what you posted
